pkg/client/buildconfigs.go

```diff
@@ -73,7 +74,7 @@ func (c *buildConfigs) Get(name string) (result *buildapi.BuildConfig, err error
 func (c *buildConfigs) WebHookURL(name string, trigger *buildapi.BuildTriggerPolicy) (*url.URL, error) {
 	switch {
 	case trigger.GenericWebHook != nil:
-		return c.r.Get().Namespace(c.ns).Resource("buildConfigs").Name(name).SubResource("webhooks").Suffix(trigger.GenericWebHook.Secret, "generic").URL(), nil
+		return c.r.Get().Namespace(c.ns).Resource("buildConfigs").Name(name).SubResource("webhooks").Suffix(trigger.GenericWebHook.Secret, "generic", strconv.FormatBool(trigger.GenericWebHook.AllowEnv)).URL(), nil
```

Comment:
I had interpreted the earlier comment ("you still want to look where i pointed because we do dump out the urls of the webhooks. Adding the value of this field to that output seems reasonable to me.") as meaning I had to somehow reflect whether `allowEnv` was true or false in the URL that was returned here.

If correct in that assumption, I had tried inserting it after the secret name, but before "generic", originally but that messed up the `oc start-build --from-webhook` parsing.

So do I have to include whether `allowEnv` is either true or false here in some fashion (and if it is a position thing, figure out how to make sure start-build still works)?  Or is something else needed?